tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-zip-commons/src/main/java/org/apache/tika/detect/zip/DefaultZipContainerDetector.java

```diff
@@ -36,10 +37,11 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
 import java.util.List;
 
 public class DefaultZipContainerDetector implements Detector {
-
+    
```

Comment:
Please undo not needed changes, in order to make a potential TIKA pull request more successful